src/query/frontend/ast/cypher_main_visitor.cpp

```diff
@@ -2220,6 +2252,7 @@ antlrcpp::Any CypherMainVisitor::visitAtom(MemgraphCypher::AtomContext *ctx) {
     auto *where = std::any_cast<Where *>(ctx->filterExpression()->where()->accept(this));
     return static_cast<Expression *>(storage_->Create<None>(ident, list_expr, where));
   } else if (ctx->REDUCE()) {
+    // ctx->reduceExpression()->variable()
```

Comment:
Why is this commented?

Reply:
This is a leftover from learning how to work with the ANTLR AST, thanks for letting me know about it!

Reply:
Removed in #931